### ISSUE TITLE
[NBug] SSH key file could not be found

### ISSUE THREAD

**Issue by osltest** [gitextensions/gitextensions#10609]
### Environment

- Git Extensions 4.0.1.15887
- Build f2567dea28bbf0e2cbcd2a95d4c7200e47d0d671
- Git 2.39.0.windows.2
- Microsoft Windows NT 10.0.19044.0
- .NET 6.0.12
- DPI 96dpi (no scaling)

### Issue description

''

```
GitUI.NBugReports.UserExternalOperationException: SSH key file could not be found
 ---> System.IO.FileNotFoundException: SSH key file could not be found
   --- End of inner exception stack trace ---
   at GitUI.Infrastructure.PuttyHelpers.ThrowIfFileNotFound(String filePath, String errorMessage, String heading)
   at GitUI.Infrastructure.PuttyHelpers.StartPageantIfConfigured(Func`1 sshKeyFileLoader)
   at GitUI.CommandsDialogs.FormPush.StartPageant(String remote)
   at GitUI.CommandsDialogs.FormPush.PushChanges(IWin32Window owner)
   at GitUI.CommandsDialogs.FormPush.PushClick(Object sender, EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)
```

Tried to push a commit, Putty is running and working.

### Steps to reproduce

This started after installing the latest version. Since then I cannot push to any remote repositiry, doing a manual "git push" works. I have tried reinstalling Git Extentions but this has no helped.

### Did this work in previous version of GitExtensions?

Yes. I don't know the exact version number, but it was 4 something.

**Comment by gerhardol** [comment]
Duplicate of https://github.com/gitextensions/gitextensions/issues/10530
Included in 4.0.2

**Comment by psivanov** [comment]
It's not fixed in 4.0.2.
I get the same error on

- Git Extensions 4.0.2.15986
- Build b063d3258751505e45a0ea207778a19f4914fde7
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET 6.0.12
- DPI 96dpi (no scaling)
- Microsoft.WindowsDesktop.App Version

**Comment by mstv** [comment]
This is not *the* 4.0.2 but a development build, which unfortunately does not contain the fix yet - but https://ci.appveyor.com/project/gitextensions/gitextensions/builds/45842693/artifacts do.

**Comment by RussKie** [comment]
> It's not fixed in 4.0.2. I get the same error on
> 
> * Git Extensions 4.0.2.15986
> * Build [b063d32](https://github.com/gitextensions/gitextensions/commit/b063d3258751505e45a0ea207778a19f4914fde7)

This commit is three commits before the fix.

**Comment by osltest** [comment]
It's fixed for me with 4.0.2.16028. Thanks!

**Comment by VitoMilanese** [comment]
> This is not _the_ 4.0.2 but a development build, which unfortunately does not contain the fix yet - but https://ci.appveyor.com/project/gitextensions/gitextensions/builds/45842693/artifacts do.

For me it resolves the problem. Thanks.
